README.md

```diff
@@ -1,7 +1,7 @@
-# purescript-media-types
+# Media Types
 
-[![Latest release](http://img.shields.io/github/release/purescript-contrib/purescript-media-types.svg)](https://github.com/purescript-contrib/purescript-media-types/releases)
-[![Build status](https://travis-ci.org/purescript-contrib/purescript-media-types.svg?branch=master)](https://travis-ci.org/purescript-contrib/purescript-media-types)
+[![CI](https://github.com/purescript-contrib/purescript-media-types/workflows/CI/badge.svg?branch=main)](https://github.com/purescript-contrib/purescript-media-types/actions?query=workflow%3ACI+branch%3Amain)
+[![Release](http://img.shields.io/github/release/purescript-contrib/purescript-media-types.svg)](https://github.com/purescript-contrib/purescript-media-types/releases)
 [![Pursuit](http://pursuit.purescript.org/packages/purescript-media-types/badge)](http://pursuit.purescript.org/packages/purescript-media-types/)
 [![Maintainer: garyb](https://img.shields.io/badge/maintainer-garyb-lightgrey.svg)](http://github.com/garyb)
 [![Maintainer: thomashoneyman](https://img.shields.io/badge/maintainer-thomashoneyman-lightgrey.svg)](http://github.com/thomashoneyman)
```

Comment:
These have changed in color to ‘teal’ (contrib-updater will do this automatically)

Reply:
Fixed after running `contrib-updater`.